maas-schemas/schemas/environments/environments.json

```diff
@@ -48,13 +48,14 @@
         "id": { "$ref": "#/definitions/environmentId" },
         "api": { "$ref": "https://schemas.maas.global/environments/apis.json#/definitions/apiUrl" },
         "apis": { "$ref": "https://schemas.maas.global/environments/apis.json#/definitions/apiConfigs" },
+        "tenants": { "$ref": "https://schemas.maas.global/environments/tenants.json#/definitions/tenantIndex" },
```

Comment:
I think all stages should support all tenants. Maui already has a dropdown for stages. I'm planning to add a separate dropdown for tenants.